tests/connect_redirect/connect_redirect_tests.cpp

```diff
@@ -127,16 +208,50 @@ _initialize_test_globals()
     REQUIRE((v6_addresses == 0 || v6_addresses == 3));
     IN6ADDR_SETLOOPBACK((PSOCKADDR_IN6)&_globals.addresses[socket_family_t::IPv6].loopback_address);
 
+    _globals.mode = _get_execution_mode(_execution_mode_string);
+    _globals.user_token = _logon_user(_user_name, _password);
```

Comment:
```suggestion
    _globals.user_token = _log_on_user(_user_name, _password);
```
Since I think you're using "log on" as a verb, not "logon" as a noun or adjective.